site-config.example/defaults.yml

```diff
@@ -1,75 +1,103 @@
-#       - 200:250
-#     reservable_vlan_ranges:
-#       - 250:300
-#     bridge_name: br-em1
-#     external_interface: em1
-#   - name: physnet2
-#     bridge_name: br-em2
-#     external_interface: em2
-#   - name: public
-#     bridge_name: br-ex
-#     external_interface: em3
-#     # This should be your public IP block assigned to your deployment.
-#     cidr: 0.0.0.0/32
+# MANDATORY: Define IPv4 network address for Ironic's internal
+#   provisioning subnet; this should be an address in the same
+#   network assigned to "eno3" but NOT be the same as any address
+#   the OS configuration actually binds to the NIC
+# NOTE: This IP must reside in the same subnet as defined in
+#   ironic_dnsmasq_dhcp_range
+##kolla_internal_vip_address: 172.16.0.10
 
+###########################################################################
+# IMPORTANT: For the next two items, the IP address should reverse-resolve
+#   to the FQDN, and the FQDN should resolve to the IP
 
-# Ironic network config
-# Uncomment to enable provisioning vlan for Ironic
-# ironic_provisioning_network_vlan: 200
-# ironic_dnsmasq_dhcp_range: 10.51.0.0/24
+# MANDATORY: Define IPv4 network address for Chameleon's public
+#   web interface; this should be an address in the same network
+#   assigned to "eno1", but should NOT be the same as any address
+#   the OS configuration actually binds to the NIC
+##kolla_external_vip_address: 1.2.3.4
 
-# Physical Switch config for Neutron-ML2:
-# Uncomment with info for neutron managed switch
-# switch_configs:
-#   - name: leafswitch1
-#     device_type: netmiko_dell_force10
-#     address: 10.0.0.10
-#     auth:
-#       username: chameleon
-#       password: "{{ dell_switch_password }}"
-#     snmp:
-#       community_string: "{{ dell_switch_community_string }}"
-#     ngs_config:
-#       persist_changes: False
-#       fast_cli: True
+# MANDATORY: Define the FQDN of the Chameleon host
+# NOTE: If using TLS, then this must match the FQDN of the certificate
+##kolla_external_fqdn: chi-xyz.example.tld
+###########################################################################
 
-# If you are allowing users to reserve public IPs,
-# this should correspond to the Neutron ID of your
-# public (gateway) network. It defaults to allowing public/Public
-# blazar_floatingip_reservation_network_regex: "[pP]ublic"
+###########################################################################
+# Identity Federation Settings
+# If you are operating Chameleon as an Independent Testbed, then leave the
+#   following settings unchanged
+# Otherwise, enable the appropriate federation service
+enable_keystone_federation: no
+enable_keystone_federation_openid: no
+keystone_idp_client_id: null
+###########################################################################
+
+###########################################################################
+# MANDATORY: Neutron networks
+# At minimum, you MUST define:
+#   0) A "public" network, corresponding to "eno1"
```

Comment:
nit: it may not always be eno4, I'm wondering how best to communicate what this value should be. For the public network, it needs to be on an access interface (i.e., not trunked) that can reach the Internet w/o NAT. For the private network, it should be on an interface that can reach the top-of-rack switches, both for bare metal and for KVM configurations.

@msherman64 any suggestions?